### ISSUE TITLE
Can I set a default for fencedCodeBlockHighlighting in Markdown mode?

### ISSUE THREAD

**Issue by jackdouglas** [codemirror/CodeMirror#6283]
The Markdown mode [ depends on other modes for properly highlighted code blocks](https://codemirror.net/mode/markdown/index.html), which works brilliantly.

fencedCodeBlockHighlighting only specifies "Whether to syntax-highlight fenced code blocks, if given mode is included" — my question is whether there is any way to get a default syntax-highlight mode for plain fences (` ``` ` rather than ` ```whatever `)?

If there isn't any current way would you be interested in a patch to enable this as a mode configuration option? It would be a one-liner something like changing:

```
      state.localMode = modeCfg.fencedCodeBlockHighlighting && getMode(match[2]);                
```

to 

```
      state.localMode = modeCfg.fencedCodeBlockHighlighting && getMode(match[2] || modeCfg.defaultLang);                
```

**Comment by jackdouglas** [comment]
`defaultLang` probably isn't the best choice of option name but naming things isn't my forte

**Comment by marijnh** [comment]
Yes, a PR for that would be welcome. Maybe call it `fencedBlockDefaultMode`?

**Comment by jackdouglas** [comment]
Thanks, please see https://github.com/codemirror/CodeMirror/pull/6285

I did wonder whether fencedCodeBlockHighlighting instead should be overloaded to either be boolean false, or the default mode, but perhaps that is too opaque?

I've tested the code change but not the docs change.

**Comment by marijnh** [comment]
Implemented in #6285
